.github/workflows/auto-review.yml

```
@@ -0,0 +1,102 @@
+        run: |
+          python3 << 'PYEOF'
+          import os, subprocess, json, re
+
+          diff_content = open("/tmp/pr_diff.diff").read()
+          pr_number = os.environ.get("PR_NUMBER", "")
+          owner = os.environ.get("OWNER", "")
+          repo = os.environ.get("REPO", "")
+
+          review_prompt = f"""You are an expert code reviewer. Analyze the following PR diff and provide actionable feedback.
+
+PR #{pr_number} from {owner}/{repo}
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)